tests/Linting/Linters/FullyQualifiedFacadesTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Tests\Linting\Linters;
+
+use PHPUnit\Framework\TestCase;
+use Tighten\TLint\Linters\FullyQualifiedFacades;
+use Tighten\TLint\TLint;
+
+class FullyQualifiedFacadesTest extends TestCase
+{
+    /** @test */
+    public function does_not_trigger_when_file_is_not_namespaced()
+    {
+        $file = <<<file
+<?php
+
+Hash::make('test');
+file;
+
+        $lints = (new TLint)->lint(
+            new FullyQualifiedFacades($file)
+        );
+
+        $this->assertEmpty($lints);
+    }
+
+    /** @test */
+    public function does_not_trigger_on_alias_usage_without_import()
```

Comment:
oh wait.. so you're making the lint NOT trigger here, but you ARE having the formatter fix it? if the linter isn't linting it we shouldn't format it.

Reply:
We only format the import statement.  So in the example below if File was imported as `use File;` it would have been formatted as `use Illuminate\Support\Facades\File;`.

Since I have added `does_not_trigger_on_alias_usage_without_import` to both linter and formatter test I've removed this one.